src/app/components/Button/__tests__/button.test.tsx

```diff
@@ -1,12 +1,40 @@
 import * as React from 'react';
-import { render, screen } from '@testing-library/react';
+import { render, screen, fireEvent } from '@testing-library/react';
 import { CreateButton } from '../Button';
 import '@testing-library/jest-dom'
 
+// Mocking of the createAsset function
+jest.mock('../Asset/Asset', () => ({
```

Comment:
Hey @srinipandiyan thanks for the contribution. But the jest test case is failing. 

1. Inside tsconfig.json file you will need to add 'node' to the types array in the compilerOptions section. 
2. Also once you have added node to the types array,  you will encounter the error: Cannot find module '../Asset/Asset' from 'src/app/components/Button/__tests__/button.test.tsx'

I will suggest running the test locally first and check if it runs and then update the tests.

Reply:
Hey @srinipandiyan any updates here? :et us know if you have doubts

Reply:
Made the requested changes.